crates/sui-config/tests/snapshots/snapshot_tests__network_config_snapshot_matches.snap

```diff
@@ -80,6 +80,7 @@ validator_configs:
       force-disable-epoch-sui-conservation-check: false
       enable-state-consistency-check: false
       force-disable-state-consistency-check: false
+      enable-move-vm-paranoid-checks: false
```

Comment:
@lxfind does this snapshot change mean that we need to version `ExpensiveSafetyCheckConfig`, or is this fine?